pallets/subtensor/tests/children.rs

```diff
@@ -32,6 +34,7 @@ fn test_do_set_child_singular_success() {
     });
 }
 
+// 2: Attempt to set child in non-existent network
 // SKIP_WASM_BUILD=1 RUST_LOG=info cargo test --test children -- test_do_set_child_singular_network_does_not_exist --exact --nocapture
 #[test]
 fn test_do_set_child_singular_network_does_not_exist() {
```

Comment:
It is duplicated with test_do_revoke_child_singular_network_does_not_exist, test_do_set_children_multiple_network_does_not_exist, test_do_revoke_children_multiple_network_does_not_exist. just test the network not exists.